starter_code/express-spotify-app/views/albums.ejs

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Search for an Artist on Spotify</title>
+    <link rel="stylesheet" type="css/text" href="../public/stylesheets/style.css">
+  </head>
+  <body>
+    <% albums.forEach(alb => { %>
+      <img src="
+      <%alb.images[0].url%>
+      " alt="">
+      <p><%=alb.name%></p>
+      <a href="/tracks/<%=alb.id%>">View Tracks</a>
```

Comment:
This link is routing the user to "/tracks/:id" route, but you don't have a handler for that route, so it is not showing anything. You will need to add an app.get route handler for that, and make it render a view.